application/features/authentication.feature

```diff
@@ -178,36 +178,32 @@ Feature: Authentication
     Then the response status code should be 200
 
   Scenario Outline: Check profile review flag on user resource in response to authenticate call
-    Given there is a "shep_clark" user with a review_profile_after in the <tense>
+    Given there is a "shep_clark" user in the database
+      And that user has a review_profile_after in the <reviewTense>
+      And that user has a nag_for_mfa_after in the <mfaTense>
+      And that user has a nag_for_method_after in the <methodTense>
       And I provide the following valid data:
       | property  | value       |
       | username  | shep_clark  |
       | password  | govols!!!   |
     When I request "/authentication" be created
     Then the following data is returned:
-      | property       | value      |
-      | employee_id    | 123        |
-      | profile_review | <yesorno>  |
+      | property       | value        |
+      | employee_id    | 123          |
+      | profile_review | <review>     |
+      | mfa.add        | <mfaAdd>     |
+      | method.add     | <methodAdd>  |
 
     Examples:
-      | tense    | yesorno  |
-      | past     | yes      |
-      | present  | yes      |
-      | future   | no       |
-
-  Scenario: Check profile review flag after an authenticate call
-    Given there is a "shep_clark" user with a review_profile_after in the past
-      And I provide the following valid data:
-        | property  | value       |
-        | username  | shep_clark  |
-        | password  | govols!!!   |
-      And I request "/authentication" be created
-    When I request "/user/123" be retrieved
-    Then the response status code should be 200
-      And the following data is returned:
-        | property       | value      |
-        | username       | shep_clark |
-        | profile_review | no         |
+      | mfaTense | methodTense | reviewTense | mfaAdd | methodAdd | review |
```

Comment:
it's great to see the truth table expanded in this way, you've got a very complex situation here now.  This does not cover all possibilities though, is that because you were able to eliminate a bunch for logical reasons?

Reply:
What combination did I miss? There are eight combinations of three binary variables (2^3 = 8).

Reply:
`yes | yes | yes` for example?

Reply:
Since `mfaAdd`, `methodAdd`, and `review` are outputs, that's not a valid test case.

Reply:
ok